collector/pkg/component/analyzer/cpuanalyzer/config.go

```diff
@@ -29,5 +36,7 @@ func NewDefaultConfig() *Config {
 		JavaTraceSlowTime:     500,
 		SegmentSize:           40,
 		EdgeEventsWindowSize:  2,
+		JavaTraceDeleteInterval: 10,
+		JavaTraceExpirationTime: 30,
```

Comment:
I would recommend setting the default value of JavaTraceExpirationTime to 120 seconds (2 minutes), considering that we allow traces to come in within this timeframe.